src/win32/mmap.h

```diff
@@ -13,16 +13,10 @@
 #define MAP_ANONYMOUS (1 << 5)
 #define MAP_FAILED    ((void *)-1)
 
-#ifdef __cplusplus
-extern "C" {
-#endif
 
```

Comment:
```suggestion
```

Reply:
Addressed in commit da3d9e3f1e1a6d116df5830cf506f30ff56196b3.